mongodbatlasv2/api_x509_authentication.go

```diff
@@ -332,6 +340,13 @@ type X509AuthenticationApiListDatabaseUserCertificatesRequest struct {
 	itemsPerPage *int32
 	pageNum *int32
 }
+type X509AuthenticationApiListDatabaseUserCertificatesQueryParams struct {
+		groupId string
+		username string
+		includeCount *bool
+		itemsPerPage *int32
+		pageNum *int32
```

Comment:
fields are private so they can't be used

Reply:
Moving to draft. Sorry for early ping